modules/tensor_mechanics/doc/content/modules/tensor_mechanics/1d_elastic_waves.md

```diff
@@ -0,0 +1,98 @@
+
+Other applications of frequency domain dynamics are: (1) computation of band structure (dispersion curves) of lattices/metamaterial, (2) inverse design for vibration control, e.g. design a system so that it has as minimum/maximum response at particular frequencie/s, (3) material properties inversion/optimization given discrete responses.
+
+On the other hand, frequency domain analyses can be advantageous over time domain counterpart in several cases, for example, when the frequency spectrum of a signal consists of a few frequencies, or, when it is needed to have a better control of the numerical dispersion.
+
+# Problem Description
+
+A a uniaxial beam, of length L=1, fixed on the left. On the right boundary, the displacement boundary condition is a cosine function, see [cantilever].
+
+!media media/tensor_mechanics/Cantilever_beam.png style=width:60%; caption=2D cantilever problem with a prescribed displacement boundary condition on the right end. id=cantilever
+
+The PDE that mathematically describes this time-harmonic structural dynamics problem is as follows:
+\begin{equation}
+    -\nabla \cdot \sigma +\rho \frac{\partial^2 u}{\partial t^2}=0 \label{eq1}
+\end{equation}
+
+For the boundary condition, we apply the sommerfeld radiation condition on the left, and a harmonic source (cosine) on the right as follows:
+\begin{equation}
+    \sqrt E \, \nabla u= i\sqrt \rho \, \frac{\partial u}{\partial t} \, \qquad at \, x=0,
+    \\
+    u= A \, cos(2 \pi f t) \, \qquad at \, x=L,\label{eq2}
+\end{equation}
+where $f$ is the frequency, $A=0.5$, and  $2 \pi f=\omega=10$.
+
+In equation 1, we consider Young's modulus and the density as constants and equal to unity.
+Equation (1) can be written as:
+\begin{equation}
+     -\nabla \cdot \sigma + \frac{\partial^2 u}{\partial t^2} = 0 \label{eq3}
+\end{equation}
+
+Now, we consider the plane wave $u= r e^{i(\omega t - k x)}$ is a solution for equation (3), with $r$ being the complex amplitudes, and $k$ is the wave number and $i=\sqrt{-1}$. "hint: you can verify that $r e^{i(\omega t - k x)}$ is a solution by direct substitution in equation (3)", $\Longleftrightarrow$ $k^2= \frac{\omega^2}{c_p^2}$, where $k$ is the wave number, and $c_p=\sqrt \frac{E}{\rho}$ is the longitudinal wave velocity. As we noted before that Young's modulus and the density as constants and equal to unity, thus, $c_p=1$.
+
+Now, equation (2) and the associated BCs. and can be written as,
```

Comment:
instead of equation (2)  You need use the equation labels to update them as the write up changes.

Reply:
done, I used equation labels instead.